app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -516,7 +483,13 @@ class LuciBuildService {
     final Map<String, dynamic> exec = <String, dynamic>{'cipdVersion': cipdVersion};
 
     return ScheduleBuildRequest(
-        builderId: builderId, tags: tags, properties: properties, notify: notificationConfig, exe: exec);
+        builderId: builderId,
+        tags: processedTags,
+        properties: processedProperties,
+        notify: notificationConfig,
+        fields: 'id,builder,number,status,tags',
+        exe: exec,
+        dimensions: dimensions);
```

Comment:
nit: add trailing commas for nicer formatting

Reply:
Added